pinot-controller/src/main/java/org/apache/pinot/controller/util/FileIngestionHelper.java

```diff
@@ -176,7 +176,7 @@ public static void copyURIToLocal(Map<String, String> batchConfigMap, URI source
     if (!PinotFSFactory.isSchemeSupported(sourceFileURIScheme)) {
       PinotFSFactory.register(sourceFileURIScheme, batchConfigMap.get(BatchConfigProperties.INPUT_FS_CLASS),
           IngestionConfigUtils.getInputFsProps(IngestionConfigUtils.getConfigMapWithPrefix(
-              batchConfigMap, BatchConfigProperties.INPUT_FS_PROP_PREFIX)));
+              batchConfigMap, BatchConfigProperties.INPUT_FS_PROP_PREFIX + IngestionConfigUtils.DOT_SEPARATOR)));
```

Comment:
How is this related to adding an adhoc task api?  It is better to keep bug fixes independent.

Reply:
this is a bug identified during the code copy.
It generated the props all started with DOT_SEPARATOR

Reply:
Isnt it better to keep this in a separate PR ? (that way, when we cut releases, it is easy to identify a bug fix vs feature)

Reply:
https://github.com/apache/pinot/pull/8511